include/module.modulemap

```diff
@@ -0,0 +1,8 @@
+framework module CrashReporter {
```

Comment:
Use links like other files in this directory, no need to copy.

BTW why it should be in `include`?

Reply:
I tried other places in test branches but have no success. Also, I have found during investigation that people were faced with the error _modulemap '/some/path/module.modulemap' should be inside the 'include' directory_.